src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/BuildPropertyPage.xaml

```diff
@@ -156,14 +156,14 @@
   <!-- TODO create fwlink for this HelpUrl -->
   <BoolProperty Name="Deterministic"
                 DisplayName="Deterministic"
-                Description="Indicates whether the compiler should produce identical assemblies for identical inputs."
+                Description="Compilation produces identical output for identical inputs."
                 HelpUrl="https://docs.microsoft.com/dotnet/csharp/language-reference/compiler-options/deterministic-compiler-option"
                 Category="General" />
   
   <!-- TODO create fwlink for this HelpUrl -->
   <EnumProperty Name="ErrorReport"
                 DisplayName="Internal compiler error reporting"
-                Description="Controls when internal compiler error (ICE) reports are sent to Microsoft."
+                Description="Report internal compiler error (ICE) reports to Microsoft."
```

Comment:
How about "Send internal compiler error (ICE) reports to Microsoft." then "report" isn't repeated